### PR TITLE
Api service

### PR DESCRIPTION
**Pull request by VelMakarova**

### CODE REVIEW

File: ui/src/api/index.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import AuthApi from './auth-api';
+import HealthCheckApi from './healthcheck-api';
+
+export default class ApiService {
+  static getAuthApi = () => AuthApi;
```
Comment (y-andriievskyi) [inline ui/src/api/index.ts:5]: I'd probably use a single static method (e.g. getService) that accepts a key and use a Map of services that can be accessed by a passed key. So we won't have to modify the ApiService at all.

File: ui/src/api/auth-api.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import RestClient from './rest-client';
+
+export default class AuthApi {
+  static createUser(url: string, data: { email: string; password: string }) {
```
Comment (y-andriievskyi) [inline ui/src/api/auth-api.ts:4]: Use an Interface for that.
In the future we will need a shared package to share DTOs across client/server apps

File: ui/src/api/auth-api.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    return RestClient.post(url, data);
+  }
+
+  static fetchUser(url: string) {
```
Comment (y-andriievskyi) [inline ui/src/api/auth-api.ts:8]: I don't like that we pass the url as a param here. I think it should be specified on a service level

File: ui/src/api/fetch-func/user.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import ApiService from '../index';
+import Paths from '../paths';
+
+export async function getUser() {
+  const res = await ApiService.getAuthApi().fetchUser(Paths.USER);
```
Comment (y-andriievskyi) [inline ui/src/api/fetch-func/user.ts:5]: [2] I don't like that we pass the url as a param here. I think it should be specified on a service level
Might wanna operate with userId here

File: ui/src/api/paths.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+enum Paths {
+  HEALTH = '/health',
+  USER = 'https://jsonplaceholder.typicode.com/users/1',
```
Comment (y-andriievskyi) [inline ui/src/api/paths.ts:3]: why JPH?)

**Review by NatalySheludko — APPROVED** [review]

**Review by y-andriievskyi — APPROVED** [review]

**Pull request merged**